python-package/xgboost/training.py

```diff
@@ -332,6 +338,27 @@ def cv(params, dtrain, num_boost_round=10, nfold=3, metrics=(),
     -------
     evaluation history : list(string)
     """
+    if early_stopping_rounds is not None and len(metrics) > 1:
+        raise ValueError('Check your params.'\
+                                     'Early stopping works with single eval metric only.')
+
+    if early_stopping_rounds is not None:
+        sys.stderr.write("Will train until cv error hasn't decreased in {} rounds.\n".format(\
```

Comment:
You might want to combine the above if statements for `early_stopping_rounds` and then have an inner if statement for `len(metrics) > 1`. Also wrap the `sys.stderr.write` around with `if verbose_eval` like I did in [this commit](15a0d27).